### ISSUE TITLE
Log statements should be preceded and followed by a blank line

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#332]
A call to a log method should be followed by a blank line if the following line contains a call to something that is no log method.
The reason is ease of reading.

Following should report a violation:
```c#
Log.Debug("Initializing");
var x = 42;
var y = "something";
var z = Guid.NewGuid();
```

While following should **not** report a violation:
```c#
Log.Debug("Initializing");

var x = 42;
var y = "something";
var z = Guid.NewGuid();
```

**Comment by RalfKoban** [comment]
Log methods should also have a preceding blank line.


Following should report a violation:
```c#
var x = 42;
var y = "something";
var z = Guid.NewGuid();
Log.Debug("Initializing");
```

While following should **not** report a violation:
```c#
var x = 42;
var y = "something";
var z = Guid.NewGuid();

Log.Debug("Initializing");
```
